qm-plugins/qm-db-connections/qm-db-connections.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Plugin Name: Query Monitor DB Connections
+ * Description: Additional collector for Query Monitor for DB connection information.
+ * Version: 1.0
+ * Author: Automattic, Rebecca Hum
+ */
+
+if ( ! defined( 'VIP_GO_APP_ENVIRONMENT' ) ) {
+	return;
+}
+
+add_action( 'plugins_loaded', 'register_qm_db_connections' );
+function register_qm_db_connections() {
+	if ( ! class_exists( 'QM_Collectors' ) ) {
+		return;
+	}
+
+	require_once __DIR__ . '/class-qm-db-connections-collector.php';
+
+	QM_Collectors::add( new QM_DB_Connections_Collector() );
```

Comment:
Pardon the ignorance, Is this idiomatic for a QM_Collector to be added even though we may not know that a user is logged in? In other words, do we need to check here if a user is logged in as to not add any extra overhead for no benefit?

Reply:
You know what. I think we could actually move the entire block https://github.com/Automattic/vip-go-mu-plugins/blob/90bd992af476e75c21cc075c012c031f79aeafd7/query-monitor.php#L157-L168 into `wpcom_vip_qm_require()` so that the capability filter can catch it at https://github.com/Automattic/vip-go-mu-plugins/blob/90bd992af476e75c21cc075c012c031f79aeafd7/query-monitor.php#L84-L86